samples/FunctionApp/FunctionApp.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.3.0" OutputItemType="Analyzer" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.7.0" />
```

Comment:
Not 100% sure, but we *might* need to keep `OutputItemType` - vaguely remember last year that we needed to make changes to this app to test the host and worker e2e ([PR here](https://github.com/Azure/azure-functions-dotnet-worker/pull/616/files)). One of the changes was adding this property. If you have time, could you test that e2e debugging works without `OutputItemType` before merging? I can also assist here if needed.

Reply:
Let's sync about this offline!

Reply:
Validated that we can still debug with the host without `OutputItemType`

![Screen Shot 2022-09-28 at 2 44 39 PM](https://user-images.githubusercontent.com/2198905/193365257-3b0d927c-2dd8-4ee4-9236-a83693630a8b.png)